<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>891</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page0">
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLineEdit" name="id_lineedit">
         <property name="text">
          <string>king</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="login_btn">
         <property name="text">
          <string>로그인</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="pw_lineedit">
         <property name="text">
          <string>1234</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="move_signup_page_btn">
         <property name="text">
          <string>회원가입</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>280</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디와 비밀번호가 기억이 안나신다구요?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>당신은 공부할 자격이 없습니다.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="pushButton1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(200, 200, 200);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(70, 70, 70);
border-style: solid;
border-radius: 20px}</string>
      </property>
      <property name="text">
       <string>학습</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>370</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(200, 200, 200);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(70, 70, 70);
border-style: solid;
border-radius: 20px}</string>
      </property>
      <property name="text">
       <string>문제풀기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>370</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(200, 200, 200);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(70, 70, 70);
border-style: solid;
border-radius: 20px}</string>
      </property>
      <property name="text">
       <string>점수확인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>370</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(200, 200, 200);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(70, 70, 70);
border-style: solid;
border-radius: 20px}</string>
      </property>
      <property name="text">
       <string>QnA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton5">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>370</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
border-style: solid;
border-radius: 20px
}
QPushButton:hover{
background-color: rgb(200, 200, 200);
border-style: solid;
border-radius: 20px}

QPushButton:pressed{
background-color: rgb(70, 70, 70);
border-style: solid;
border-radius: 20px}</string>
      </property>
      <property name="text">
       <string>상담</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>461</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1~3등만 표시하는 그래프 x축은 이름, y축은 점수</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>241</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>빠따선생님.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>801</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QLabel" name="quiz_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quiz_lineedit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="quiztable">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>31</y>
        <width>440</width>
        <height>420</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string>문제</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>쓴 답</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>정답</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>채점</string>
       </property>
      </column>
     </widget>
     <widget class="QLCDNumber" name="correct_answer">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>330</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="incorrec_answer">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>370</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>330</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>맞힌 개수</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>370</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>틀린 개수</string>
      </property>
     </widget>
     <widget class="QLabel" name="OXlabel">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>260</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="add_quiz_btn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>420</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>문제추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_quiz_btn">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>420</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>저장하기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_quiz_btn">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>420</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>불러오기</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="quiz_counter">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>290</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>290</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>푼 문제 수</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="all_quiz_counter">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>250</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>250</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>총 문제 수</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>120</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>내 등수 / 전체 수</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>문제 맞힌 갯수 / 푼 갯수</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>어떻게 배치할지 생각 중</string>
      </property>
     </widget>
     <widget class="QPushButton" name="movetomainbtn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>420</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>461</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="graph_layout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page5">
     <widget class="QTableWidget" name="qna_tablewidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>851</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>학생 이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>질문</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>답변</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="qna_name_input">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="qna_back_btn">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>440</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qna_send_btn">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>440</y>
        <width>89</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>질문 등록</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qna_refresh_btn">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>440</y>
        <width>89</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>새로고침</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qna_question_input">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>480</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>이름 입력</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>질문 입력</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page6">
     <widget class="QLineEdit" name="make_id_line">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="make_pw_line">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>211</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="check_pw_line">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>242</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="make_name_line">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>313</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="make_mail_line">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>344</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>141</y>
        <width>42</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>212</y>
        <width>56</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>243</y>
        <width>87</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호 확인</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>314</y>
        <width>28</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>345</y>
        <width>56</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>이메일</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_check_btn">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>143</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>중복확인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signup_btn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>390</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="id_check_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pw_check_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page7">
     <widget class="QTextBrowser" name="counsel_chat_box">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>701</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="counsel_input_chat">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>400</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="send_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>440</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>보내기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>440</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>나가기</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="rank_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>순위 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="rank_value_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>점수 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>이름이름</string>
    </property>
   </widget>
   <widget class="QLabel" name="hi_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>님 반갑습니다. 좋은 하루 되세요.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backbtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>임시버튼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_out_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>로그아웃</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="score_counter">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
